CONTRIBUTORS.md

```diff
@@ -20,4 +20,5 @@ Community
 [@dgoeke]: https://github.com/dgoeke
 [@sparkofreason]: https://github.com/sparkofreason
 [@bfontaine]: https://github.com/bfontaine
-[@sunilgunisetty]: https://github.com/sunilgunisetty
\ No newline at end of file
+[@sunilgunisetty]: https://github.com/sunilgunisetty
+[@bartuka]: https://github.com/wandersoncferreira
```

Comment:
You should also add a line like line 13. so that your name will be displayed as well